workspaces/chrome-extension-hello-world/hard-mode.js

```diff
@@ -0,0 +1,39 @@
+console.log('hard-mode.js Running');
```

Comment:
This PR is amazing 😄 
Regarding the formatting -- are you able to run `yarn format` and `yarn lint` from within the `workspaces/chrome-extension-hello-world` directory?